xml/book_cloud_deploy_crowbar.xml

```diff
@@ -53,12 +53,9 @@
   <title>Setting Up Non-&ostack; Services</title>
   <xi:include href="depl_nostack_services.xml"/>
  </part>
- <part xml:id="part.depl.maintenance">
-  <title>Maintenance and Support</title>
-  <xi:include href="depl_maintenance.xml"/>
-  <xi:include href="depl_logfiles.xml"/>
+ <part xml:id="part.depl.troubleshooting">
+  <title>Troubleshooting and Support</title>
   <xi:include href="depl_troubleshooting.xml"/>
```

Comment:
There is also two instances of the same target issues in the _depl_troubleshooting_ file: depl_troubleshooting.xml:24:53: error: IDREF "cha.deploy.logs" without matching ID